server/rest/src/main/java/org/infinispan/rest/server/operations/mediatypes/printers/HTMLOutputPrinter.java

```diff
@@ -0,0 +1,32 @@
+package org.infinispan.rest.server.operations.mediatypes.printers;
+
+import java.nio.charset.Charset;
+import java.util.Iterator;
+
+import org.infinispan.CacheSet;
+import org.infinispan.rest.server.operations.exceptions.ServerInternalException;
+import org.infinispan.rest.server.operations.mediatypes.OutputPrinter;
+
+public class HTMLOutputPrinter implements OutputPrinter {
+
+   @Override
+   public byte[] print(String cacheName, CacheSet<?> keys) {
+      StringBuilder sb = new StringBuilder("<html><body>");
+      Iterator<?> it = keys.iterator();
```

Comment:
You could be fancy and do something like:
`keys.stream().map(b -> Escaper.escapeHtml(b)).map(s -> String.format("<a href=\"%s/%s\">%s</a><br/>", cacheName, s, s)).collect(Collectors.joining("<html><body>", "", "</body></html>");`